scripts/check-stack2nix.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+# Get relative path to script directory
+set -xe
+scriptDir=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+
+time nix-build -A stack2nix -o $scriptDir/stack2nix -Q -j 4
+
+time $scriptDir/stack2nix/bin/stack2nix --revision 4bcf1e6b601d531f753ebdb6aec23e19d4b08e6c https://github.com/input-output-hk/cardano-sl.git > $scriptDir/default-2.nix
```

Comment:
I was hoping we could just run `pkgs/generate.sh` and check `git diff` to have no output. Then we wouldn't need this script at all. Am I missing something with that approach?